sources/iTermApplicationDelegate.m

```diff
@@ -1478,6 +1484,10 @@ - (IBAction)checkForIncompatibleSoftware:(id)sender {
     }
 }
 
+- (IBAction)checkForUpdatesFromMenu:(id)sender {
+    [[[SUUpdater alloc] init] checkForUpdates:(sender)];
```

Comment:
Is this kosher? We will have two SUUpdater objects. I think it's better to use the one that already exists, which is an object in MainMenu.xib. Add an IBOutlet to iTermApplicationDelegate so you can call it from here.